app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable name="category" type="String" />
+        <variable name="isSelected" type="Boolean" />
+        <variable name="callbacks" type="org.cru.godtools.ui.dashboard.tools.CategoryAdapterCallbacks"/>
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:minHeight="64dp"
+            android:layout_margin="4dp"
+            android:padding="@dimen/tools_card_corner_radius"
+            android:shadowColor="@color/tintable"
+            tools:cardBackgrounColor="@color/gray_80"
+            app:cardBackgroundColor="@{isSelected ? @color/gray_80 : @color/white }"
+            app:cardCornerRadius="@dimen/card_corner_radius"
+            android:onClick="@{() -> callbacks.onCategorySelected(category)}"
+            app:cardElevation="4dp">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
```

Comment:
is there a reason you are using `ConstraintLayout` here and not just putting the `TextView` directly in the `CardView`?